src/hook_manager.hh

```diff
@@ -133,6 +140,7 @@ private:
     struct HookData;
 
     SafePtr<HookManager> m_parent;
+    Vector<SafePtr<HookManager>> m_linked;
```

Comment:
I am unsure about exactly how and when `SafeCountable`/`SafePtr` should be used. Here it is just done like this to match `m_parent`